assemblies/webpackage/src/main/resources-filtered/common-ui/package.json

```diff
@@ -371,6 +371,14 @@
           }
         }
       },
+      "pentaho/visual/models/Treemap": {
```

Comment:
Just realized that, in this same module, `webpackage`, the RequireJS changes related with bundles also need to be duplicated on...
https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/0b17995811f7c316c801315b06e445347516deec/assemblies/webpackage/src/main/resources-filtered/common-ui/preconfig.js#L386